unlocks.csv

```diff
@@ -389,13 +389,13 @@ Analysis|LaserCutter|0,0,LaserCutter
 Goal|OnCraftBattery|0,1,Battery
 Goal|OnCraftComputerChip|0,1,ComputerChip|CopperWire
 Goal|OnCraftTank|0,1,Tank
-Goal|OnCraftDoubleTank|0,1,DoubleTank|Glass|Tank
-Goal|UnlockHatchingEnzymes|0,1,HatchingEnzymes
-Goal|UnlockRadiationSuit|0,1,
-Goal|UnlockRadiationSuit|1,1,
-Goal|UnlockRadiationSuit|2,1,
-Goal|Aurora_RingRoom_Terminal2|0,1,Welder|LaserCutter|Battery
-Goal|Aurora_RingRoom_Terminal3|0,1,Welder|LaserCutter|Battery
+Goal|OnCraftDoubleTank|0,0,DoubleTank|Glass|Tank
+Goal|UnlockHatchingEnzymes|0,0,
+Goal|UnlockRadiationSuit|0,0,
+Goal|UnlockRadiationSuit|1,0,
+Goal|UnlockRadiationSuit|2,0,
+Goal|Aurora_RingRoom_Terminal2|0,0,Welder|LaserCutter|Battery
+Goal|Aurora_RingRoom_Terminal3|0,0,Welder|LaserCutter|Battery
```

Comment:
The above 5 just means not having to wait for the explosion - radiation suit being tedious for glitched, and the others being tedious for glitchless. Aurora might still be required for glitchless PRAWN.